src/twisted/protocols/ftp.py

```diff
@@ -2593,6 +2627,7 @@ def lineReceived(self, line):
         (Private) Parses the response messages from the FTP server.
         """
         # Add this line to the current response
+        line = line.decode(self._encoding)
```

Comment:
This is OK for Python 3, but on Python 2 this is now making line a ```unicode``` on Python 2, when before it was a ```str```.  This might not be what we want.  Can you change this to:

```
if isinstance(line, bytes) and bytes != str:
    line = line.decode(self._encoding)
```